src/App.js

```diff
@@ -1,15 +1,25 @@
 import React from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import { genreList } from "./components/Genre/genreList";
 import MovieListPage from './components/MovieListPage/movieListPage';
-import './App.css';
+import MovieDetailsWrapper from './components/MovieDetails/movieDetailsWrapper';
 
 const selectGenre = (index) => {
```

Comment:
redundant function, it'd be better to remove it (as well as passing it through props everywhere it's used)

Reply:
Done.